### DISCUSSION TITLE
Unable access to wp-admin

### DISCUSSION THREAD

**Discussion by Pascal26120** [rhubarbgroup/.github#48] [Support (ROC)] [answered]
When I active redis-cache plugging in wordpress, I can not access to mysite.com/wp-admin
## Steps to Reproduce

1. Install a wordpress site
2. Install redis on debian server
3. Install redis cache on wordpress
4. Active redis
5. Try to access mysite.com/wp-admin

## Environment

* Plugin version: 2.4.4
* PHP version:7.4.33
* WordPress version:6.3.1

**Comment by tillkruss** [comment]
Please post:

1. Your PHP error logs
2. The status code of a failed `/wp-admin` request
3. The HTML output of a failed `/wp-admin` request
4. The browser console output of a failed `/wp-admin` request

**Comment by Pascal26120** [comment, marked as answer]
Hi,

Thank you for reply.
I correct the issue by adding the following line in my wp-config.php : 

`define('WP_REDIS_IGNORED_GROUPS', ['user_meta', 'users', 'userlogins', 'usermeta']);`

Regards
